devel/ubertooth/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           conflicts_build 1.0
+PortGroup           github 1.0
+PortGroup           legacysupport 1.1
+
+github.setup        greatscottgadgets ubertooth 3e9de91f406ea13f4a5a76c3fd5e21c2a0375ad8
+version             2022-07-19
+revision            0
+categories          devel net security
+description         Software, firmware and hardware designs for Ubertooth.
+long_description    Project Ubertooth is an open source wireless development platform suitable \
+                    for Bluetooth experimentation. Ubertooth ships with a capable BLE (Bluetooth Smart) \
+                    sniffer and can sniff some data from Basic Rate (BR) Bluetooth Classic connections.
+license             GPL-2
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+
+homepage            https://github.com/greatscottgadgets/ubertooth
+
+checksums           rmd160  3a3545670f44bcb3a1ce257bf5dd5da42fba1014 \
+                    sha256  3ad1dc36accd8b7f69fdf4489df93afcaa4d1e7bdd9448b24a23eacc525deec5 \
+                    size    1307090
+
+# clock_gettime
+legacysupport.newest_darwin_requires_legacy 15
+
+patchfiles          patch-python3.diff
+
+set python_branch   3.10
+set python_ver      [string map {. {}} ${python_branch}]
+configure.python    ${prefix}/bin/python${python_branch}
+
+depends_build-append \
+                    port:pkgconfig
+
+if {${build_arch} ni [list ppc ppc64]} {
+    depends_build-append \
+                    port:python${python_ver} \
+                    port:py${python_ver}-numpy \
+                    port:py${python_ver}-pyside \
+                    port:py${python_ver}-setuptools
+}
+
+depends_lib-append  path:${prefix}/lib/pkgconfig/libbtbb.pc:libbtbb \
+                    path:${prefix}/lib/pkgconfig/libusb-1.0.pc:libusb
+
+post-patch {
+    fs-traverse f ${worksrcpath}/host/python {
+        if {[file extension ${f}] eq ".py"} {
+            reinplace -q "s|/usr/bin/env python3|${configure.python}|" ${f}
+        }
+    }
+    reinplace "s|/usr/bin/env python3|${configure.python}|" ${worksrcpath}/host/python/ubtbr/ubertooth-btbr
+    reinplace "s|/usr/bin/env python3|${configure.python}|" ${worksrcpath}/host/python/specan_ui/ubertooth-specan-ui
+    reinplace "s|/usr/bin/env python3|${configure.python}|" ${worksrcpath}/host/python/specan_ui/setup.py.in
+    reinplace "s|@PYTHON3@|${configure.python}|g" ${worksrcpath}/host/python/ubtbr/CMakeLists.txt
+}
+
+cmake.source_dir    ${worksrcpath}/host
+cmake.module_path   ${cmake.source_dir}/cmake/modules
+
+conflicts_build     ${name}
```

Comment:
why is this the case - that's very annoying and should be resolved...

Reply:
> why is this the case - that's very annoying and should be resolved...

Follows from instructions in README: If previous versions of libbtbb or Ubertooth tools are installed they should be removed before building the latest release. https://github.com/greatscottgadgets/ubertooth/blob/master/host/README.md

Reply:
@reneeotten The rest are corrected, do you think we do not need a conflict here, Macports gonna take care of the thing?
I know there are ports using such setting though.

Reply:
I will try now and update.

Reply:
@reneeotten No issues:
```
macmini:~ svacchanda$ port -v installed ubertooth
The following ports are currently installed:
  ubertooth @2022-07-16_0 requested_variants='' platform='darwin 10' archs='ppc' date='2022-11-05T19:48:18+0800'
  ubertooth @2022-07-19_0 (active) requested_variants='' platform='darwin 10' archs='ppc' date='2022-11-05T20:18:57+0800'
```
Upgraded with self-conflict removed.